README.md

```diff
@@ -1,88 +1,142 @@
-# crds-styles
+crds-styles
+==========
 
-`crds-styles` provides the primary CSS framework and global style patterns that power [crossroads.net](http://crossroads.net) and it's related properties. Any elements and/or implementation details included within the [Digital Design Kit (DDK)](http://github.com/crdschurch/crds-styleguide) are provided by this NPM package. Any deviations from the patterns defined there should be implemented in the client application.
+`crds-styles` provides the primary CSS framework and global style patterns that
+power [crossroads.net](http://crossroads.net) and it's related properties.
```

Comment:
🚨 Grammar error: Should be 'its' because it's possessive, not a contraction of 'it is'

Reply:
Thank you for explaining that to me.